test/TestUtil.ts

```diff
@@ -113,6 +115,16 @@ export class TestUtil {
         await GlobalState.update(this.GLOBAL_STATE);
     }
 
+    static async storeEnableLocalFabric(): Promise<void> {
+        this.ENABLE_LOCAL_FABRIC = await vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_LOCAL_FABRIC);
+        console.log('Storing enable Local Fabric:', this.ENABLE_LOCAL_FABRIC);
```

Comment:
Maybe change the message to 'Storing enable Local Fabric **to settings** ' to be similar to the restoring message.